### PR TITLE
Update spark pipeline instructions.

### PR DESCRIPTION
**Pull request by deflaux**
Fixes https://github.com/googlegenomics/readthedocs/issues/42 by directing users to Dataproc.

### CODE REVIEW

File: docs/source/use_cases/compute_principal_coordinate_analysis/1-way-pca.rst [hunk 1]
```diff
@@ -86,20 +86,23 @@ The following command will run PCA over the BRCA1 region within the :doc:`/use_c
...
-    --master spark://hadoop-m:7077 \
-    /PATH/TO/googlegenomics-spark-examples-assembly-1.0.jar \
-    --client-secrets /PATH/TO/YOUR/client_secrets.json \
+    googlegenomics-spark-examples-assembly-1.0.jar \
     --variant-set-id 3049512673186936334 \
     --references chr17:41196311:41277499 \
     --output-path gs://YOUR-BUCKET/output/platinum-genomes-brca1-pca.tsv
 
-The above command line runs the job over a small portion of the genome, only taking a few minutes.  If modified to run over a larger portion of the genome or the entire genome, it may take a few hours depending upon how many machines are in the Spark cluster.
+The above command line runs the job over a small portion of the genome, only taking a couple minutes.  If modified to run over a larger portion of the genome or the entire genome, it may take a few hours depending upon how many machines are in the Spark cluster.
```
Comment (mbookman) [inline docs/source/use_cases/compute_principal_coordinate_analysis/1-way-pca.rst]: A "couple" is two. Does it consistently take two minutes?

File: docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst [hunk 2]
```diff
@@ -46,20 +46,23 @@ The following command will run a two-way PCA over the BRCA1 region within the :d
...
-    --master spark://hadoop-m:7077 \
-    /PATH/TO/googlegenomics-spark-examples-assembly-1.0.jar \
-    --client-secrets /PATH/TO/YOUR/client_secrets.json \
+    googlegenomics-spark-examples-assembly-1.0.jar \
     --variant-set-id 10473108253681171589 3049512673186936334 \
     --references 17:41196311:41277499 chr17:41196311:41277499 \
     --output-path gs://YOUR-BUCKET/output/two-way-brca1-pca.tsv
 
-The above command line runs the job over a small portion of the genome, only taking a few minutes.  If modified to run over a larger portion of the genome or the entire genome, it may take a few hours depending upon how many machines are in the Spark cluster.
+The above command line runs the job over a small portion of the genome, only taking a couple minutes.  If modified to run over a larger portion of the genome or the entire genome, it may take a few hours depending upon how many machines are in the Spark cluster.
```
Comment (mbookman) [inline docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst]: A "couple" is two. Does it consistently take two minutes?

**Comment by mbookman** [comment]
Just a couple of comments, otherwise LGTM.

File: docs/source/use_cases/compute_principal_coordinate_analysis/1-way-pca.rst (hunk 1 above)
Comment (deflaux) [inline docs/source/use_cases/compute_principal_coordinate_analysis/1-way-pca.rst]: Actually, it took 22 seconds last time I ran it :-)

File: docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst (hunk 2 above)
Comment (deflaux) [inline docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst]: This takes a little longer, but still < 2 minutes.

**Pull request merged**